src/vanilla.ts

```diff
@@ -104,6 +104,10 @@ export const proxy = <T extends object>(initialObject: T = {} as T): T => {
           const errorOrPromise = (value as any)[PROMISE_ERROR] || value
           Object.defineProperty(snapshot, key, {
             get() {
+              if (PROMISE_RESULT in (value as any)) {
+                return (value as any)[PROMISE_RESULT]
+              }
+
```

Comment:
```suggestion
```

This is just a stylistic preference.
I know it's too packed currrently and having some empty lines would help, but not here (or not only here).